examples/benchmarks/Stacking/README.md

```diff
@@ -0,0 +1 @@
+# Stacking
```

Comment:
Please add more descriptions about it.
You can refer to the examples to others.